doc/release-notes.rst

```diff
@@ -508,6 +508,149 @@ Notable Changes
 * rocksdb: update to latest (Xiaoxi Chen)
 * rpm: loosen ceph-test dependencies (Ken Dreyer)
 
+v0.94.4 Hammer (draft)
+======================
+
+This Hammer point release ...
```

Comment:
This Hammer point fixes several important bugs in Hammer, as well as fixing interoperability issues that are required before an upgrade to Infernalis.  That is, all users of earlier version of Hammer or any version of Firefly will first need to upgrade to hammer v0.94.4 or later *before* upgrading to Infernalis (or future releases).